### PR TITLE
Date and cross icon issue with customer profile.

### PR DESCRIPTION
**Pull request by papnoisanjeev**
## Issue Reference
#6471

### CODE REVIEW

**Comment by Sanjaybhattwebkul** [comment]
@papnoisanjeev 
**Not fixed**, the icon should be in the right corner of the input box, please find the screenshot [Here](https://prnt.sc/sPy-h2LuJtTC)

**Comment by papnoisanjeev** [comment]
@Sanjaybhattwebkul 

Updated the date icon position at right corner.

**Comment by Sanjaybhattwebkul** [comment]
@papnoisanjeev 
**Still not fixed** for RLT, please find the screenshot [Here](https://prnt.sc/MG2JY2402NT_)

**Comment by Sanjaybhattwebkul** [comment]
@papnoisanjeev 
In the case of RTL still i am getting the same issue,

**Review by Sanjaybhattwebkul — APPROVED** [review]

**Comment by Sanjaybhattwebkul** [comment]
@papnoisanjeev 
Now this is **working fine,** please find the screenshot [Here](https://prnt.sc/PoYoUDG73WuQ)

**Comment by Sanjaybhattwebkul** [comment]
@papnoisanjeev 
Please resolve the **conflicts**.

File: packages/Webkul/Velocity/src/Resources/assets/sass/components/rtl.scss [hunk 1]
```diff
@@ -574,16 +574,14 @@ body {
...
             }
-        } 
-
-        .rango-delete {
-            float: right !important;
-            margin-left: 4px !important;
+        }
+        
+        #date-of-birth:after {
+            right: 1142px !important;
```
Comment (jitendra-webkul) [inline packages/Webkul/Velocity/src/Resources/assets/sass/components/rtl.scss:584]: This property will cause issues in big screen

File: packages/Webkul/Velocity/src/Resources/assets/sass/components/rtl.scss [hunk 2]
```diff
@@ -574,16 +574,14 @@ body {
             }
         }
 
-        .d-inline-block  {
-            .material-icons {
-                float: right !important;
-                margin-left: 4px !important;
+        #date-of-birth {
+            .cross-icon {
+                right: 1120px !important;
```
Comment (jitendra-webkul) [inline packages/Webkul/Velocity/src/Resources/assets/sass/components/rtl.scss:579]: This property will cause issues in big screen

File: packages/Webkul/Velocity/src/Resources/assets/sass/components/UI.scss [hunk 3]
```diff
@@ -87,17 +87,28 @@
...
+    position: relative;
+
+    .cross-icon {
+        position: absolute;
+        right: 34px;
+        top: 8px;
+    }
+}
+
 #date-of-birth:after{
```
Comment (jitendra-webkul) [inline packages/Webkul/Velocity/src/Resources/assets/sass/components/UI.scss:101]: Move :after to parent selector

**Review by Sanjaybhattwebkul — APPROVED** [review]

**Review by Sanjaybhattwebkul — APPROVED** [review]

File: packages/Webkul/Velocity/src/Resources/assets/sass/components/rtl.scss [hunk 4]
```diff
@@ -586,16 +586,16 @@ body {
...
+                left: 34px;
             }
-        } 
-
-        .rango-delete {
-            float: right !important;
-            margin-left: 4px !important;
+        }
+        
+        #date-of-birth:after {
```
Comment (jitendra-webkul) [inline packages/Webkul/Velocity/src/Resources/assets/sass/components/rtl.scss:596]: We can move this under parent #date-of-birth selector

**Review by jitendra-webkul — APPROVED** [review]

**Pull request merged**